ultralytics/engine/predictor.py

```diff
@@ -375,6 +375,9 @@ def save_predicted_images(self, save_path="", frame=0):
 
         # Save images
         else:
+            if save_path.split("/")[-1].startswith("predict"):
```

Comment:
Could you please replace this check with `save_path[-1]=='/'`? It's more short. 😊